database/migrations/2023_12_12_171400_alert_rule_note.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('alert_rules', function (Blueprint $table) {
+            $table->text('notes')->default(null);
```

Comment:
This should be
`$table->text('notes')->->nullable();` 
(and it will default to null without need to specify it)